sdl_android/src/main/java/com/smartdevicelink/api/screen/SoftButtonManager.java

```diff
@@ -215,17 +224,17 @@ protected SoftButtonObject getSoftButtonObjectById(int buttonId) {
      * Set softButtonObjects list and upload the images to the head unit
      * @param softButtonObjects the list of the SoftButtonObject values that should be displayed on the head unit
      */
-    protected void setSoftButtonObjects(List<SoftButtonObject> softButtonObjects) {
+    protected void setSoftButtonObjects(@NonNull CopyOnWriteArrayList<SoftButtonObject> softButtonObjects) {
```

Comment:
Might as well wait until now to do the conversion from `List` to `CopyOnWriteArrayList`